common/mqtt_client/tests/packet_size_tests.rs

```diff
@@ -41,7 +41,7 @@ async fn packet_size_within_limit() -> Result<(), anyhow::Error> {
 async fn packet_size_exceeds_limit() -> Result<(), anyhow::Error> {
     // Start the broker
     let mqtt_server_handle = tokio::spawn(async {
-        start_broker_local("../../configuration/rumqttd/rumqttd_5884.conf").await
+        rumqttd_broker::start_broker_local("../../configuration/rumqttd/rumqttd_5884.conf").await
```

Comment:
Don't use relative paths here. Just put the config file along the tests.

```suggestion
        rumqttd_broker::start_broker_local("tests/rumqttd_5884.conf").await
```